tests/ui/perfherder/alerts_test.jsx

```diff
@@ -226,6 +245,31 @@ const mockModifyAlert = {
 const mockUpdateAlertSummary = (alertSummaryId, params) => ({
   failureStatus: null,
 });
+const alertsView = () =>
+  render(
+    <AlertsView
+      user={testUser}
+      projects={repos}
+      location={{
+        pathname: '/alerts',
+        search: '',
+      }}
+      history={createMemoryHistory('/alerts')}
+      frameworks={[
```

Comment:
I won't block on this, but what I meant in my previous comment is that you could store this object in the `frameworks` variable instead and then map over it, returning only the names in a new array.

Reply:
Thanks, more elegant this way! You don't even care about how many `frameworks` are available. No indices used, no coupling.